onnx/defs/schema.cc

```diff
@@ -786,8 +786,16 @@ void OpSchema::BuildFunction(FunctionProto& function_body, const std::vector<Ope
     function_body.add_attribute(a.first);
   }
 
-  for (auto& relied_opset : relied_opsets) {
-    *(function_body.mutable_opset_import()->Add()) = relied_opset;
+  if (!relied_opsets.empty()) {
+    for (auto& relied_opset : relied_opsets) {
```

Comment:
why do we need a default here? I am not convinced we need this

Reply:
After discussion, the original behaviour in ONNX is correct. Instead, ORT should specify what the relied_opsets are for the function op. Close this PR now.